core/roboconf-agent/src/test/java/net/roboconf/agent/internal/AgentMessageProcessor_ImportsTest.java

```diff
@@ -61,9 +63,16 @@
 
 	@Before
 	public void initializeAgent() throws Exception {
+		final MessagingClientFactoryRegistry registry = new MessagingClientFactoryRegistry();
```

Comment:
Any chance to put this code in a static method somewhere?
I see it is used to least 3 times (if not more) in the agent tests. That would avoid reading the same code and comments, over and over again.